src/utils/components/CPUMemory/CPUMemory.tsx

```diff
@@ -19,17 +22,15 @@ const CPUMemory: FC<CPUMemoryProps> = ({ vm }) => {
   const { t } = useKubevirtTranslation();
   const itMatcher: V1InstancetypeMatcher = vm?.spec?.instancetype;
   const { instanceType, instanceTypeLoaded, instanceTypeLoadError } = useInstanceType(itMatcher);
+  const isVMRunning = vm?.status?.printableStatus === printableVMStatus.Running;
```

Comment:
there is a function for it in a general util

Reply:
I will take a look, thanks. But it is interesting that this line of the code is in so many places of our codebase, and everybody is fine with that, just here it is not fine.

Reply:
I added this funciton last week, because I see too many repeat code such as this, so this is the first time I we can use it other the pr created this funciton

Reply:
We need a lot of refactoring in our codebase :)